### ISSUE TITLE
GPG Signatures & Hashes

### ISSUE THREAD

**Issue by 2SI3NX** [WikidPad/WikidPad#25]
Please can you start signing and posting hashes of the release files?
As well as your public key.
This would greatly increase security and some users feel safer that way.

**Comment by benkinooby** [comment]
Agree! In fact, there is much more to do in terms of security than just signing & hashing the release files.

Before we can tackle this, we first need to sort out the canonical distribution path of WikidPad - currently https://wikidpad.sourceforge.net/ is still mentioned in many places.

I suggest to keep this issue open for now.
